pkg/inserter/common_inserter.go

```diff
@@ -65,34 +66,59 @@ func (c *CommonInserter) tryParseDate(item map[string]interface{}) map[string]in
 	return item
 }
 
+func (c *CommonInserter) replaceMultiRefs(src string, reg *regexp.Regexp) string {
+	ms := reg.FindAllString(src, 10)
+	if len(ms) == 0 {
+		return ""
+	}
+	for _, m := range ms {
+		refID := strings.TrimSuffix(strings.TrimPrefix(m, "#{"), "}")
```

Comment:
```suggestion
		refID := m[2 : len(m)-1]
```
これで良い気がする
https://go.dev/play/p/oL8Cx_CtWbL

Reply:
あー？

Reply:
なるほど、普通にこう書けるのか